libs/algo/odc/algo/_percentile.py

```diff
@@ -0,0 +1,71 @@
+from typing import List, Sequence, Tuple
+import dask.array as da
+import xarray as xr
+import numpy as np
+from ._geomedian import reshape_for_geomedian
+from ._masking import keep_good_np
+
+
+def np_percentile(xx, percentile, nodata, high=True):
```

Comment:
default should be `high=False`, but also I don't like boolean flags like that. You have two modes that you want to support

- `nodata` is smallest possible values
- `nodata` is largest possible value

Reply:
do you reckon this would be better as two seperate functions?

Reply:
no, I prefer an "enumeration" type for this option even if it only has 2 possible values to a boolean flag. Like say
`mode = 'nodata_lowest'` or something like that. Cause eventually you'll want `mode = "auto"` and with bool you'll have to do weird things like `None|True|False`, but with named mode it's just one more added to the list.

Reply:
oh actually I might as well just make this `auto`